components/expenses/ExpenseInfoSidebar.js

```diff
@@ -83,7 +83,11 @@ const ExpenseInfoSidebar = ({ isLoading, host, collective, children }) => {
         </Container>
       </Box>
       {children && <Box my={50}>{children}</Box>}
-      <ExpandableExpensePolicies host={host} collective={collective} mt={50} />
+      <ExpandableExpensePolicies
+        host={host}
+        organization={collective?.type === 'PROJECT' ? collective.parent : collective}
```

Comment:
- Events have parents too, filtering on the type is too restrictive
- We only want to use the parent policies if the children (aka, project or event) does not define one